rct229/ruletest_engine/ruletest_jsons/ashrae9012019/SHW/rule_11_17.json

```diff
@@ -17703,7 +17703,11 @@
                                                 "floor_name": "Floor1",
                                                 "spaces": [
                                                     {
-                                                        "id": "Space 1"
+                                                        "id": "Space 1",
+                                                        "service_water_heating_uses": [
+                                                            "SHW 1"
```

Comment:
@jugonzal07 - With the updated logic, I had to also update 11-17-(c) and (d). If these make sense to you, could you update the RPD spreadsheet?